src/compose/ComposeBox.js

```diff
@@ -423,6 +448,22 @@ class ComposeBox extends PureComponent<Props, State> {
   }
 }
 
+const getPmrecipientIsDeactivated = (state: GlobalState, narrow: Narrow): boolean => {
+  if (!isPrivateNarrow(narrow)) {
+    return false;
+  }
+  const recipientEmail = narrow[0].operand;
+  return !getUserIsActive(state, recipientEmail);
+};
+
+const getGroupPmrecipientIsDeactivated = (state: GlobalState, narrow: Narrow): boolean => {
```

Comment:
Nit: I'd capitalize `Recipient` in `getPmRecipientIsDeactivated` and `getGroupPmRecipientIsDeactivated`

Reply:
Fixed.